src/universal/components/Button/Button.js

```diff
@@ -76,7 +77,7 @@ const Button = props => {
     buttonOptions.push(styles.isBlock);
   }
 
-  const buttonStyles = combineStyles.apply(null, buttonOptions);
+  buttonStyles = combineStyles.apply(null, buttonOptions);
```

Comment:
Ah, looking at the entire file, I think you could rewrite this to be clearer. Like so:

```javascript
  const buttonOptions = [styles.base, styles[size], styles[styleThemeName]];
  const buttonTitle = title || label;
  const themeName = upperFirst(theme);
  const styleThemeName = `${style}${themeName}`;

  if (disabled) {
    buttonOptions.push(styles.disabled);
  }

  if (isBlock) {
    buttonOptions.push(styles.isBlock);
  }

  const buttonStyles = combineStyles.apply(...buttonOptions);
```